nacossync-worker/src/main/java/com/alibaba/nacossync/timer/CheckRunningStatusAllThread.java

```diff
@@ -0,0 +1,124 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.nacossync.timer;
+
+import com.alibaba.nacos.api.naming.NamingService;
+import com.alibaba.nacos.api.naming.pojo.ListView;
+import com.alibaba.nacos.client.naming.utils.CollectionUtils;
+import com.alibaba.nacossync.cache.SkyWalkerCacheServices;
+import com.alibaba.nacossync.constant.MetricsStatisticsType;
+import com.alibaba.nacossync.constant.TaskStatusEnum;
+import com.alibaba.nacossync.dao.TaskAccessService;
+import com.alibaba.nacossync.extension.holder.NacosServerHolder;
+import com.alibaba.nacossync.monitor.MetricsManager;
+import com.alibaba.nacossync.pojo.model.TaskDO;
+import com.google.common.eventbus.EventBus;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ *  when the database task service name is empty, check all the services in the cluster and create a synchronization task.
+ * @ClassName: CheckRunningStatusAllThread
+ * @Author: ChenHao26
+ * @Date: 2022/7/20 10:30
+ * @Description: muti sync data
+ */
+@Slf4j
+public class CheckRunningStatusAllThread implements Runnable{
+    
+    private MetricsManager metricsManager;
+    
+    private SkyWalkerCacheServices skyWalkerCacheServices;
+    
+    private TaskAccessService taskAccessService;
+    
+    private EventBus eventBus;
+    
+    private NacosServerHolder nacosServerHolder;
+    
+    private FastSyncHelper fastSyncHelper;
+    
+    public CheckRunningStatusAllThread(MetricsManager metricsManager, SkyWalkerCacheServices skyWalkerCacheServices,
+            TaskAccessService taskAccessService, EventBus eventBus, NacosServerHolder nacosServerHolder,
+            FastSyncHelper fastSyncHelper) {
+        this.metricsManager = metricsManager;
+        this.skyWalkerCacheServices = skyWalkerCacheServices;
+        this.taskAccessService = taskAccessService;
+        this.eventBus = eventBus;
+        this.nacosServerHolder = nacosServerHolder;
+        this.fastSyncHelper = fastSyncHelper;
+    }
+    
+    /**
+     * 根据ns级别进行数据同步
+     */
+    @Override
+    public void run() {
+        Long startTime = System.currentTimeMillis();
+        try {
+            List<TaskDO> taskDOS = taskAccessService.findServiceNameIsNull()
+                    .stream().filter(t -> t.getStatus() == null || t.getStatus() == 0)
+                    .collect(Collectors.toList());
+    
+            for (TaskDO taskDO : taskDOS) {
+                List<String> serviceNameList = getServiceNameList(taskDO);
+                if (CollectionUtils.isEmpty(serviceNameList)) {
+                   continue;
+                }
+                
+                //如果是null，证明此时没有处理完成
+                List<String> filterService = serviceNameList.stream()
+                        .filter(s -> skyWalkerCacheServices.getFinishedTask(taskDO.getTaskId() + s ) == null)
```

Comment:
The field `s` should have a meaningful name. such as serviceName

Reply:
> The field `s` should have a meaningful name. such as serviceName

done